lua/wikis/commons/Infobox/Extension/Achievements.lua

```diff
@@ -67,21 +73,22 @@ end
 ---@param player string
 ---@param onlySolo boolean
 ---@param playerLimit integer
----@return string
+---@return ConditionTree
 function Achievements._playerConditions(player, onlySolo, playerLimit)
 	player = player:gsub(' ', '_')
 	local playerNoUnderScore = player:gsub('_', ' ')
 
 	if onlySolo then
-		return '([[opponentname::' .. player .. ']] OR [[opponentname::' .. playerNoUnderScore .. ']])'
+		return ConditionUtil.anyOf(ColumnName('opponentname'), {player, playerNoUnderScore}) --[[@as ConditionTree]]
```

Comment:
since we use the `{player, playerNoUnderscore}` several times in this function could make it a var